src/main/scala/com.gu.contentapi.client/model/Queries.scala

```diff
@@ -142,6 +142,7 @@ trait FilterExtendedParameters[Owner <: Parameters[Owner]] extends Parameters[Ow
   def leadContent = StringParameter("lead-content")
   def fromDate = DateParameter("from-date")
   def toDate = DateParameter("to-date")
+  def contentType = StringParameter("type")
```

Comment:
I am note sure it makes a lot of sense to have the parameter on `ItemQuery` but that the same for `from-date` and `to-date` so it is a minor issue that should be addressed separately.